### DISCUSSION TITLE
How to change the page that is redirected after logging in

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4847] [Plugins]
When I log in with a regular account, I go to the project page, how do I modify it to go to the dashboard?
And when I log in with my SAML Google account, I get to my work time page, how do I modify it to go to the dashboard?

**Comment by 1yuukuu** [comment]
Settings->Users->Profile->Edit->Preferences->Initial view after login (2 from last in the list) you will get a drop-down to select from. Hope this helps.

**Reply by seleondevel** [reply]
Thanks to you, I solved it. Thank you.
